pkg/controller/servicediscovery/servicediscovery_controller.go

```diff
@@ -140,8 +142,8 @@ func (r *ReconcileServiceDiscovery) Reconcile(request reconcile.Request) (reconc
 		return reconcile.Result{}, err
 	}
 
-	ligthhouseConfigMap := newLigthhouseConfigMap(instance)
-	if _, err = helpers.ReconcileConfigMap(instance, ligthhouseConfigMap, reqLogger,
+	ligthhouseDnsConfigMap := newLigthhouseDnsConfigMap(instance)
```

Comment:
This would be the ideal time to fix the “ligth” typo on both sides ;-).

Reply:
Done. There were more than just 2 :D